runtime-tests/index.ts

```diff
@@ -0,0 +1,35 @@
+import { ArgumentParser } from 'argparse';
+
+const config = {
+	chain: 'polkadot',
+};
+
+const argv = process.argv.slice(0, 2);
+
+/**
+ * The arg parser takes in two commands. 
+ * @arg --chain The chain to be passed into the jest test
+ * @arg --config The path to the correct jest config. This is important as the
+ * jest config inside of /runtime-tests ignores all the other tests.
+ */
+const parser = new ArgumentParser();
+
+parser.add_argument('--chain', { choices: ['polkadot', 'kusama', 'westend'], default: 'polkadot' });
+parser.add_argument('--config', { default: './runtime-tests/jest.config.js' });
+
+const args = parser.parse_args();
+
+// Set the chain property for the jest test
+config['chain'] = args.chain
+
+// Pass in the Jest CLI path option to the correct config
+argv.push('--config=' + args.config)
+
+// Store configuration on env
+process.env.__CONFIGURATION = JSON.stringify(config);
+
+// Setting real ARGV
+process.argv = argv;
```

Comment:
Is this a normal pattern? Makes sense but I haven't seen it before

Reply:
I believe for passing in custom cli args into a jest instance its been used by many, but I am not sure it's a common pattern.